pycytominer/cyto_utils/cells.py

```diff
@@ -262,24 +371,50 @@ def get_subsample(self, df=None, compartment="cells", rename_col=True):
         self.is_subset_computed = True
 
     def load_compartment(self, compartment):
+        """Creates the compartment dataframe.
+
+        Parameters
+        ----------
+        compartment : str
+            The compartment to process.
+
+        Returns
+        -------
+        pandas.core.frame.DataFrame
+            Compartment dataframe.
+
+        """
         compartment_query = "select * from {}".format(compartment)
         df = pd.read_sql(sql=compartment_query, con=self.conn)
         return df
 
     def aggregate_compartment(
-        self, compartment, compute_subsample=False, aggregate_args=None
+        self,
+        compartment,
+        compute_subsample=False,
+        compute_counts=False,
+        aggregate_args=None,
     ):
         """Aggregate morphological profiles. Uses pycytominer.aggregate()
 
-        :param compartment: string indicating the specific compartment, defaults to "cells"
-        :type compartment: str
-        :param compute_subsample: determine if subsample should be computed, defaults to False
-        :type compute_subsample: bool
-        :param aggregate_args: additional arguments passed as a dictionary as input to pycytominer.aggregate()
-        :type aggregate_args: None, dict
-        :return: Aggregated single-cell profiles
-        :rtype: pd.DataFrame
+        Parameters
+        ----------
+        compartment : str, default "cells"
+            Compartment to aggregate.
+        compute_subsample : bool, default False
+            Whether or not to subsample.
+        compute_counts : bool, default False
```

Comment:
Currently two different kinds of counts are being computed - site count and object count. So I grouped them together. Would it be better if I separate them into `compute_site_count` and compute_object_count` instead of `compute_counts`? It feels like separating them if better as they may be other counts we would want to compute in the future

Reply:
oh I see, it's probably ok for now as is